client/src/commons/NoContextElements/HebrewTextField.tsx

```diff
@@ -16,6 +16,7 @@ const HebrewTextField: HebrewTextFieldType = (props) => {
   return (
     <TypePreventiveTextField
         {...props}
+        disabled={props.disabled}
```

Comment:
isn't {...props} already spreading props.disabled?

Reply:
you right, removed it :)